src/ResourceManager/ManagedServiceIdentity/Commands.ManagedServiceIdentity/help/Get-AzureRmUserAssignedIdentity.md

```diff
@@ -0,0 +1,223 @@
+---
+external help file: Microsoft.Azure.Commands.ManagedServiceIdentity.dll-Help.xml
+Module Name: AzureRM.ManagedServiceIdentity
+online version: 1.0.0
+schema: 2.0.0
+---
+
+# Get-AzureRmUserAssignedIdentity
+
+## SYNOPSIS
+Gets User Assigned Identity/identities.
+
+## SYNTAX
+
+### SuscriptionParameterSet (Default)
+```
+Get-AzureRmUserAssignedIdentity [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceGroupParameterSet
+```
+Get-AzureRmUserAssignedIdentity -ResourceGroupName <String> [-Name <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzureRmUserAssignedIdentity** gets existing user assigned identities.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzureRmUserAssignedIdentity -ResourceGroupName PSRG -Name ID1
+```
+
+This example cmdlet gets the User Assigned Identity with name **ID1** under the resource group **PSRG**
+
+Id                : /subscriptions/586d0246-0344-49dc-a790-59c916b0c309/resourcegroups/PSRG/providers/Microsoft.ManagedIdentity/userAssignedIdentities/ID1
```

Comment:
Can you put all of your sample outputs inside of script blocks? This is hard to read otherwise.